cypress/integration/edit_deputy.spec.js

```diff
@@ -9,6 +9,7 @@ describe("Edit deputy tab", () => {
         cy.get('#f-team').focus().clear();
         cy.get('#f-team').type("New Team Name")
         cy.get('form').submit()
+        cy.url().should("contain", "?success=teamDetails");
```

Comment:
I'd suggest removing this line. The success attribute in the URL is an implementation detail, whereas what we want to test is behaviour. If we changed how we get the banner to display (e.g. using headers, or even switching to a completely different UI framework), we'd want this test to still pass by checking that the banner is displayed.